packages/spec-parser/src/utils.ts

```diff
@@ -179,119 +189,147 @@ export class Utils {
     path: string,
     spec: OpenAPIV3.Document,
     options: ParseOptions
-  ): boolean {
-    const pathObj = spec.paths[path] as any;
+  ): APIValidationResult {
```

Comment:
This method will also check validation for custom copilot scenario?

Reply:
Yes, same above, currently for all